<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gbExchangeRates">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Exchange Rates</string>
   </property>
   <widget class="QComboBox" name="cbbSymbol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSymbol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspRate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblRate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbArchivedExchangeRates">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download archived rates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownloadExchangeRates">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtExchangeRateDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="displayFormat">
     <string>dd MMMM yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>781</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>-10</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
